dss_datamover/client_application.py

```diff
@@ -405,6 +406,21 @@ def message_server_index(self):
             except Exception as e:
                 self.logger.excep("Monitor-Index - {}".format(e))
 
+        while len(nfs_mount_retry_list) > 0:  # Retry failed NFS mounts for a period of time. 
+            retry_message, start_time = nfs_mount_retry_list.pop(0)
```

Comment:
just pop() should be good enough. No pop(0) required

Reply:
pop() will pop the last one in the list, and if its retry failed, it will append to the list tail again, so I think pop(0) here is necessary.